src/test/scala/chiselTests/LiteralExtractorSpec.scala

```diff
@@ -59,6 +59,20 @@ class LiteralExtractorSpec extends ChiselFlatSpec {
     bigIntFromDouble should be(bigInt53)
   }
 
+  "doubles and big decimals" should "be rounded identically" in {
+
+    for (double <- Seq(1.0, 1.1, 1.5, 1.6, 2.5)) {
```

Comment:
This has both `1.5` and `2.5` which means that this is checking to make sure it's not using `HALF_EVEN`. 👍